src/chat-client/index.ts

```diff
@@ -480,6 +480,38 @@ class Chat extends ClientBase {
     }
   }
 
+  /**
+   * Let's a conversation partner back in after they've reset their account. You can
+   * get a list of such candidates with getResetConvMembers()
+   * @param username - the username of the user who has reset
+   * @param conversationId
+   * @example
+   * await bot.chat.addResetConvMember({username: "chris", conversationId: "abc1234567"})
+   */
+  public async addResetConvMember(param: chat1.ResetConvMemberAPI): Promise<chat1.GetResetConvMembersRes> {
+    await this._guardInitialized()
```

Comment:
unrelated to this PR but
1. TS has `protected` / `private' etc so using underscore prefix isn't necessary
1. This function can throw and its not wrapped in a try so that'll bubble up (is that intended?)

Reply:
1. cool - `_guardInitialized` was already `protected`; I'm throwing JS (non-Typescript) people a bone by making our protected and private members underscore named to hint.

2. yup, intended. Most of our API calls throw their errors up if there's a problem; scripts are expected to catch them if there's an issue.